paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheetConfigurationKtx.kt

```diff
@@ -105,3 +113,55 @@ internal fun PaymentSheet.Appearance.parseAppearance() {
         )
     )
 }
+
+/**
+ * Creates a subset of the [PaymentSheet.Configuration] of values that affect the functional behavior of
+ * [PaymentSheet]. The items not included mainly affect how [PaymentSheet] will look but not affect what
+ * payment options are available to the customer:
+ * - UI elements in [PaymentSheet.GooglePayConfiguration]:
+ *   - [PaymentSheet.GooglePayConfiguration.amount]
+ *   - [PaymentSheet.GooglePayConfiguration.label]
+ *   - [PaymentSheet.GooglePayConfiguration.buttonType]
+ * - [PaymentSheet.Configuration.merchantDisplayName]
+ * - [PaymentSheet.Configuration.primaryButtonColor]
+ * - [PaymentSheet.Configuration.appearance]
+ * - [PaymentSheet.Configuration.primaryButtonLabel]
+ */
+private fun PaymentSheet.Configuration.toVolatileConfiguration(): VolatilePaymentSheetConfiguration {
```

Comment:
I don't love this approach, because it's easy to get out of date. (And it already is!) I recently added a new property (`allowsRemovalOfLastSavedPaymentMethod`), that should be in here, and it's not. It would have been really easy to miss adding it here too.